test/settings.cpp

```diff
@@ -46,6 +46,22 @@ BOOST_AUTO_TEST_CASE(settings__construct__default_context__expected)
     BOOST_REQUIRE_EQUAL(configuration.bip9_version_bit0, 1u);
     BOOST_REQUIRE_EQUAL(configuration.bip9_version_bit1, 2u);
     BOOST_REQUIRE_EQUAL(configuration.bip9_version_base, 0x20000000);
+    BOOST_REQUIRE_EQUAL(configuration.satoshi_per_bitcoin, 100000000);
+    BOOST_REQUIRE_EQUAL(configuration.initial_block_subsidy_bitcoin, 50);
+    BOOST_REQUIRE_EQUAL(configuration.recursive_money, 9999999989u);
+    BOOST_REQUIRE_EQUAL(configuration.retarget_subsidy_interval, 210000);
+    BOOST_REQUIRE_EQUAL(configuration.no_retarget_subsidy_interval, 150);
+    const auto recursive_money = configuration.max_money_recursive(
+        configuration.initial_block_subsidy_satoshi());
```

Comment:
Don't wrap test code.

Reply:
Oh, I thought that it's only code inside the `BOOST_*` macros that shouldn't be wrapped.

Reply:
https://github.com/libbitcoin/libbitcoin/wiki/Test-Guide#line-length

Reply:
Exception exists for declaring complex structures.

Reply:
For my benefit, what's the rationale for this rule?

Reply:
If you read the whole test guide there is an emphasis on declarativeness. For the same reason no conditions are allowed in the tests. Since tests are not themselves tested (infinite regression problem), it is critical that they can be evaluated for correctness visually by humans. For this reason we are strict about reducing visual clutter.

Reply:
thx :+1: